lib/grape/dsl/inside_route.rb

```diff
@@ -60,17 +71,26 @@ def declared_hash(passed_params, options, declared_params)
               else
                 memo[optioned_param_key(declared_param, options)] = passed_params[declared_param]
               end
-            else
-              declared_param.each_pair do |declared_parent_param, declared_children_params|
-                next unless options[:include_missing] || passed_params.key?(declared_parent_param)
-
-                passed_children_params = passed_params[declared_parent_param] || passed_params.class.new
-                memo[optioned_param_key(declared_parent_param, options)] = declared(passed_children_params, options, declared_children_params)
-              end
             end
           end
         end
 
+        def handle_passed_param(declared_param, passed_children_params)
```

Comment:
Probably want a `&_block` in the parameters since this yields.

Reply:
Sure, updated.